testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergSparkCompatibility.java

```diff
@@ -2924,6 +2924,114 @@ public void testSparkAlterStructColumnType(StorageFormat storageFormat)
         onSpark().executeQuery("DROP TABLE " + sparkTableName);
     }
 
+    @Test(groups = {ICEBERG, PROFILE_SPECIFIC_TESTS})
+    public void testSparkReadingTrinoBloomFilters()
```

Comment:
Either `testSparkReadingTrinoParquetBloomFilters` or add logic to run for both parquet and orc

Reply:
Let's not bring ORC tests in this PR, renaming to `testSparkReadingTrinoParquetBloomFilters` is fine